### PR TITLE
slider with incomplete functionality

### PR DESCRIPTION
**Pull request by AnikoSchmidt**
Component's functionality is imperfect: the slider moves, but not as expected.

### CODE REVIEW

**Comment by nbeliczay** [comment]
Please add screenshot or mov or gif

**Comment by AnikoSchmidt** [comment]
Hi Kali, 
Please see the screenshots to the final version. I'm also ready to submit a new PR with the week 2 assignment. 
Screen Shot 2021-11-12 at 7 28 29 PM
Screen Shot 2021-11-12 at 7 28 05 PM

File: slider/slider-component/SliderCompnent.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        let newY = beginningY+(e.offsetY-beginningY);
+        myButton.setAttribute("style", `top: ${newY}px`);
+    }
+})
+
+myButton.addEventListener('mouseup', (e) => {
+    isMouseDown = false;
+})
+
+styleYourSlider = (color, size, startPosition) => {
```
Comment (nbeliczay) [inline slider/slider-component/SliderCompnent.js:26]: for function declarations, use const:
const styleYourSlider = ...

File: slider/slider-component/SliderCompnent.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    document.getElementsByClassName("trackline")[0].style.height = size;
+    document.getElementById("slider-button").style.bottom = startPosition;
+}
+styleYourSlider('#6a040f', '700px', '0px');
+
+let array = [10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120, 130]
+// let array = [1, 2, 3]
+let labels = ''
+
+createLabels = (array) => {
```
Comment (nbeliczay) [inline slider/slider-component/SliderCompnent.js:38]: function declaration

File: slider/slider-component/SliderCompnent.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+const myButton = document.getElementById("slider-button");
+const myContainer = document.getElementById("trackercontainer");
```
Comment (nbeliczay) [inline slider/slider-component/SliderCompnent.js:2]: Stick to one naming convention. Either use camelCase or snake-case for ids

Comment (nbeliczay) [reply slider/slider-component/SliderCompnent.js:2]: or lowercase

File: slider/slider-component/SliderComponent.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        </div>
+        <div id="trackercontainer">
+          <div class="trackline">
+            <button id="slider-button">
+                -
+            </button>
+          </div>
+        </div>
+    </div>
+    <script src="./SliderCompnent.js"></script>
```
Comment (nbeliczay) [inline slider/slider-component/SliderComponent.html:20]: typo in js file name :)

**Review by nbeliczay — COMMENTED** [review]
Left a few comments on code style, but otherwise looks good!

**Review by nbeliczay — APPROVED** [review]

**Pull request merged**